docs/components/NftCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { SquareImage } from "./SquareImage";
+import { ReactNode } from "react";
+
+export function NftCard({
+  image,
+  title,
+  subtitle,
+}: {
+  image: string | undefined;
+  title: string;
+  subtitle?: ReactNode;
+}) {
+  return (
+    <>
+      <div className="nft">
+        {image && (
```

Comment:
Maybe if there is no image we can show a gray square?

Reply:
Good idea, let me do it